backends/tfhe-cuda-backend/cuda/src/integer/cast.cuh

```diff
@@ -33,4 +33,62 @@ __host__ void host_trim_radix_blocks_lsb(CudaRadixCiphertextFFI *output,
       input_start_lwe_index, input->num_radix_blocks);
 }
 
+template <typename Torus>
+__host__ uint64_t scratch_extend_radix_with_sign_msb(
+    cudaStream_t const *streams, uint32_t const *gpu_indexes,
+    uint32_t gpu_count, int_extend_radix_with_sign_msb<Torus> **mem_ptr,
+    const int_radix_params params, uint32_t num_radix_blocks,
+    uint32_t num_additional_blocks, const bool allocate_gpu_memory) {
+
+  uint64_t size_tracker = 0;
+
+  *mem_ptr = new int_extend_radix_with_sign_msb<Torus>(
+      streams, gpu_indexes, gpu_count, params, 1, num_radix_blocks,
```

Comment:
I think you can remove num_luts in int_extend_radix_with_sign_msb, just hard setting it to 1.